add_to_build_results/monodistribution/bin/cli_x86

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. $(dirname $0)/mono-env
```

Comment:
Do we want the equivalent *.bat file for windows? Or alternate shell script for Linux?